src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -268,6 +268,13 @@
     agent-shutdown-fn :- IFn
     warn-legacy-auth-conf? :- schema/Bool
     metrics-service]
+   (when (get-in raw-config [:jruby-puppet :compat-version])
+     (log/errorf "%s %s %s"
+                 "The jruby-puppet.compat-version setting is no longer supported."
+                 "To use Ruby language version 1.9.3, remove the jruby-puppet.compat-version setting."
+                 "To use a Ruby language version later than 2.0, remove the jruby-puppet.compat-version setting and add the line 'JRUBY_JAR=/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar' to the /etc/sysconfig/(pe-)puppet(-)server or /etc/default/(pe-)puppet(-)server file.")
```

Comment:
Oh right, I forgot about that about i18n... uh....... in that case maybe just forget I said anything.